src/AsmResolver.DotNet/TypeDefinition.cs

```diff
@@ -42,6 +42,81 @@ public class TypeDefinition :
         private IList<InterfaceImplementation>? _interfaces;
         private IList<MethodImplementation>? _methodImplementations;
 
+        /// <summary>
+        /// Initialize New Delegate.
```

Comment:
Grammar mistake, should include `a`.  Also avoid Camel Casing Every Word Like This. It should be a normal sentence with normal capitalization.

Example:
"Initializes a new delegate."